src/main/java/uk/gov/ons/ctp/response/casesvc/service/impl/CaseServiceImpl.java

```diff
@@ -619,12 +566,11 @@ private void effectTargetCaseStateTransition(Category category, Case targetCase)
               && !category
                   .getCategoryName()
                   .equals(CategoryDTO.CategoryName.SUCCESSFUL_RESPONSE_UPLOAD))
-          || transitionEvent == CaseDTO.CaseEvent.ACCOUNT_CREATED) {
+          || transitionEvent == CaseDTO.CaseEvent.ACTIONPLAN_CHANGED) {
         if (targetCase.getIac() != null) {
           internetAccessCodeSvcClientService.disableIAC(targetCase.getIac());
```

Comment:
We've moved the IAC disabling to party service so does this still have to be here?